src/Carbon/Lang/ca.php

```diff
@@ -98,7 +98,7 @@
     'months' => ['de gener', 'de febrer', 'de març', 'd\'abril', 'de maig', 'de juny', 'de juliol', 'd\'agost', 'de setembre', 'd\'octubre', 'de novembre', 'de desembre'],
     'months_standalone' => ['gener', 'febrer', 'març', 'abril', 'maig', 'juny', 'juliol', 'agost', 'setembre', 'octubre', 'novembre', 'desembre'],
     'months_short' => ['gen.', 'febr.', 'març', 'abr.', 'maig', 'juny', 'jul.', 'ag.', 'set.', 'oct.', 'nov.', 'des.'],
-    'months_regexp' => '/D[oD]?[\s,]+MMMM?/',
+    'months_regexp' => '/(D[oD]?[\s,]+MMMM?L{2,4}|l{2,4})/',
```

Comment:
A `|` should be added between `D[oD]?[\s,]+MMMM` and `L{2,4}|l{2,4}`.

Reply:
Thanks..

Reply:
Thank you! I just found this issue on my own and saw you were already working on it.